kimai2.yml

```diff
@@ -0,0 +1,95 @@
+captainVersion: 4
+services:
+    # kimai FPM
+    $$cap_appname-fpm:
+        depends_on:
+            - $$cap_appname-db
+        image: kimai/kimai2:$$cap_kimai_version
+        restart: 'always'
+        environment:
+            ADMINMAIL: $$cap_kimai-adminmail
+            ADMINPASS: $$cap_kimai-adminpass
+            MAILER_URL: $$cap_kimai-mailerurl
+            MAILER_FROM: $$cap_kimai-mailerfrom
+            DATABASE_URL: mysql://$$cap_mariadb-user:$$cap_mariadb-pass@srv-captain--$$cap_appname-db/$$cap_mariadb-db
+            TRUSTED_HOSTS: $$cap_appname.$$cap_root_domain,nginx,localhost,127.0.0.1
+        volumes:
+            - $$cap_appname-public:/opt/kimai/public
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    # mariadb
+    $$cap_appname-db:
+        image: mariadb:$$cap_mariadb_version
+        environment:
+            MYSQL_DATABASE: $$cap_mariadb-db
+            MYSQL_USER: $$cap_mariadb-user
+            MYSQL_PASSWORD: $$cap_mariadb-pass
+            MYSQL_ROOT_PASSWORD: $$cap_maridb-rootpass
+        volumes:
+            - $$cap_appname-db-data:/var/lib/mysql
+        restart: unless-stopped
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+    $$cap_appname:
+        image: lcxat/nginx-fpm-reverse-proxy
```

Comment:
This is using an implicit `latest` tag. Let's fix the version to prevent it from breaking the installations with future updates.

Reply:
yep, never considered to check the :latest on my own containers. Let me fix that as well.